lib/platform/github/index.js

```diff
@@ -1136,318 +1023,6 @@ function getPrBody(input) {
   );
 }
 
-// Generic File operations
-
-async function getFile(filePath, branchName) {
-  logger.trace(`getFile(filePath=${filePath}, branchName=${branchName})`);
-  if (!branchName || branchName === config.baseBranch) {
-    if (!config.fileList.includes(filePath)) {
```

Comment:
With this missing, we now hit GitHub for lots of files we don't need to (and get 404 responses). e.g. renovating a `package.json` gets this output:

```
 INFO: getFile 404 (repository=renovate-tests/preset)
       "filePath": "yarn.lock",
       "branchName": undefined
 INFO: getFile 404 (repository=renovate-tests/preset)
       "filePath": "package-lock.json",
       "branchName": undefined
 INFO: getFile 404 (repository=renovate-tests/preset)
       "filePath": "npm-shrinkwrap.json",
       "branchName": undefined
 INFO: getFile 404 (repository=renovate-tests/preset)
       "filePath": "shrinkwrap.yaml",
       "branchName": undefined
 INFO: getFile 404 (repository=renovate-tests/preset)
       "filePath": ".npmrc",
       "branchName": undefined
 INFO: getFile 404 (repository=renovate-tests/preset)
       "filePath": ".yarnrc",
       "branchName": undefined
 INFO: getFile 404 (repository=renovate-tests/preset)
       "filePath": "lerna.json",
       "branchName": undefined
```